src/org/folio/Constants.groovy

```diff
@@ -122,6 +122,10 @@ class Constants {
   static String FOLIO_HELM_V2_TEST_REPO_NAME = "folio-helm-v2-test"
   static String FOLIO_HELM_V2_TEST_REPO_URL = 'https://repository.folio.org/repository/folio-helm-v2-test/'
 
+  // Bitnami Helm Proxy Repository
+  static String BITNAMI_HELM_PROXY_REPO_NAME = "helm-bitnami-proxy"
```

Comment:
We don't need these lines, because we do not use bitnami's helm chart for BE modules, already method exists: https://github.com/folio-org/pipelines-shared-library/blob/master/vars/folioHelm.groovy#L18

Reply:
Suggested constans are removed.